third_party/dml/ci/test/TestAgent.ps1

```diff
@@ -34,9 +43,28 @@ try
 
         Write-Host "Testing $TestGroup..."
         $Results.Time.Start = (Get-Date).ToString()
-        py run_tests.py --test_group $TestGroup --tensorflow_wheel $TensorFlowWheelPath | Out-File -FilePath "test_${TestGroup}_log.txt"
-        $Results.Time.End = (Get-Date).ToString()
 
+        if ($RunOnWsl)
+        {
+            Push-Location $WslArtifactFolderAsWin
+
+            $LoadLibraryPath = wsl echo $WslArtifactFolder`:`$LD_LIBRARY_PATH
+            $WslTensorFlowWheelPath = Get-ChildItem tensorflow_directml-*-linux_x86_64.whl | Select-Object -First 1 -ExpandProperty Name
```

Comment:
At some point we really should be producing "manylinux" wheels, so let's make this glob a little more permissive to avoid headaches later.